### PR TITLE
Feat/setup database part 2

### PR DESCRIPTION
**Pull request by lanebret85**
____ Wrote Tests ____ Implemented ____ Reviewed

At this point, the rake tasks work and we are able to fill our database with all board game data from both the board_games.csv file and the BoardGameGeek API.

We still need to add validations for the BoardGame model class.
We also still need to write tests for the following items:
- BoardGame model
- BoardGamesFacade
- BoardGamesService

Necesary checkmarks:

- [ ] All Tests are Passing
- [x] The code will run locally

Type of change

- [x] New feature
- [x] Bug Fix

Implements/Fixes:

    description closes #

Check the correct boxes

- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

Testing Changes

- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

TY

### CODE REVIEW

**Review by ConnorRichmond — APPROVED** [review]
looks great.
I can work on some testing tonight as well

File: app/facades/board_games_facade.rb
Comment (reedhillmar) [inline app/facades/board_games_facade.rb:1]: This facade is a good target for future refactors

File: lib/tasks/data_load.rake
Comment (reedhillmar) [inline lib/tasks/data_load.rake:1]: This definitely belongs in the backlog, but we should build a task to update the database

**Review by reedhillmar — COMMENTED** [review]
We did it! Let's try to never do that again... I added some notes for places that would be good candidates for refactor, but not necessary for right now.

**Pull request merged**